pkg/kubelet/kubelet.go

```diff
@@ -1477,6 +1479,38 @@ func (kl *Kubelet) updateRuntimeUp() {
 	}
 }
 
+func (kl *Kubelet) reconcileCbr0Loop() {
```

Comment:
If you just want the func to run every 10 seconds, you can use util.Until(f, 10s, util.NeverStop).